cluster-service/deploy/helm/templates/azure-operators-managed-identities-config.configmap.yaml

```diff
@@ -7,6 +7,11 @@ metadata:
 data:
   azure-operators-managed-identities-config.yaml: |
     controlPlaneOperatorsIdentities:
+      control-plane-operator:
```

Comment:
This is not correct. It should be control-plane .

This is important because the names we define here *MUST* match the names specified in the spreadsheet where we define them (Proposed Key Name Changes for ARO Workload Identity) as they need to be agreed and consistent between offerings (ARO-HCP, ARO Classic, ...)